### ISSUE TITLE
DNS http://openve.org/

### ISSUE THREAD

**Issue by alexsalas** [OpenVE/openve.github.com#6]
Hola a todos, los dns del sitio apuntan al hosting del sitio personal de Daniel Rodriguez

**Comment by sadasant** [comment]
Sí, no he podido seguir con la comunidad, tampoco he querido perder el
dominio.

**Comment by alexsalas** [comment]
Seria bueno reactivar la comunidad.
Si puedes agregame @alexsalas

**Comment by sadasant** [comment]
A dónde te agrego?

Siento que tengo muchas cosas encima como para dedicarme a alguna
comunidad, estoy tratando de ayudar a AccesoLibre, realmente quiero
apoyarles. Creo que para finales de año estaré mejor. Gracias por tu
interés dude \m/

**Comment by alexsalas** [comment]
En la ORG en GitHub
https://github.com/OpenVE/
Si es como owner mejor, asi no te molesto mas xD

**Comment by foursixnine** [comment]
Yo creo que este issue lo podemos cerrar... AKA: CRYO!
